docs/feature-vote.md

```diff
@@ -1,12 +1,13 @@
 (feature-note)=
+
 # Feature voting board
 
 You can help us to prioritise development of new features by leaving a [👍 reaction](https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) on the first comment of any `enhancement` issue.
 
-Below is a list of all current enhancement issues from our core repositories,[^a] ordered by 👍
+Below is a list of all current enhancement issues from our core repositories,\[^a\] ordered by 👍
```

Comment:
broken the footnotes

Reply:
argh - hmm what should we do about these? add exceptions in the mdformat config or something?

Reply:
plugins 👇